cgp/individual.py

```diff
@@ -46,19 +89,7 @@ def crossover(self, other_parent, rng):
         """
         raise NotImplementedError("crossover currently not supported")
 
-    def _mutate(self, genome, mutation_rate, rng):
-
-        n_mutations = int(mutation_rate * len(genome.dna))
-        assert n_mutations > 0
-
-        graph = CartesianGraph(genome)
-        active_regions = graph.determine_active_regions()
-        only_silent_mutations = genome.mutate(n_mutations, active_regions, rng)
-
-        if not only_silent_mutations:
-            self.fitness = None
-
-    def mutate(self, mutation_rate, rng):
+    def mutate(self, mutation_rate: float, rng: np.random.RandomState) -> None:
         """Mutate the individual in place.
```

Comment:
```suggestion
        """Mutate the individual.
```

Reply:
In place is correct, isn't it? The function does create a new individual.

Reply:
yes, technically it's correct, but isn't it a bit redundant to state for every method that/whether it operates on the corresponding class? in contrast, if this would be a regular function that receives an individual, this additional info would be very helpful!